vault/activity_log_util_common.go

```diff
@@ -383,3 +381,28 @@ func (e *segmentReader) ReadEntity(ctx context.Context) (*activity.EntityActivit
 	}
 	return out, nil
 }
+
+func (a *ActivityLog) countsRecordToCountsResponse(record *activity.CountsRecord, includeDeprecated bool) *ResponseCounts {
```

Comment:
unfortunately `CountsRecord` and `NamespaceRecord` are in a different package, and adding receivers there would require importing the vault package, and that would cause a cycle.